manim/mobject/mobject.py

```diff
@@ -2128,12 +2128,75 @@ def set_z_index_by_z_coordinate(self):
 
 
 class Group(Mobject):
-    """Groups together multiple Mobjects."""
+    """A group of mobjects.
+
+    This can be used to group multiple :class:`~.Mobject` instances together
+    in order to scale, move, or perform other animations on them together.
+
+    To add :class:`~.Mobject`s to a :class:`~.Group`, you can either use the
+    :meth:`~.Group.add` method, or use the `+` and `+=` operators. Similarly, you
+    can subtract elements of a Group via :meth:`~.Group.remove` method, or
```

Comment:
As this is currently written, Group doesn't actually override the remove method from the class Mobject, so we must either add the method to this class, or switch this to ~.Mobject.remove. I understand that this is mainly moving VGroup code to Group, but this should also probably be done in VGroup as well (I actually made those changes anyways so this is sorta my fault in retrospect)